package.json

```diff
@@ -36,6 +36,8 @@
   },
   "scripts": {
     "postinstall": "gulp build",
+    "build": "gulp build",
+    "test": "gulp test",
```

Comment:
This is great - would you mind adding one more:
`"prestart" : "gulp build",`

So that when people start, they have all of the latest build files?